### DISCUSSION TITLE
是不是忘记添加 `.gitignore` 了

### DISCUSSION THREAD

**Discussion by pizeroLOL** [Class-Widgets/Class-Widgets#173] [常规]
https://github.com/Class-Widgets/Class-Widgets/blob/7d1b45ecce94bf4feab7942cdc779bd627f0ce69/.idea/dataSources.xml#L1-L12

**Comment by RinLit-233-shiroko** [comment]
emm这个是其他人上传的，马上删

**Comment by pizeroLOL** [comment]
ref https://github.com/Class-Widgets/Class-Widgets/commit/f610252f2de52f1ea4e3d9f0fec3951f1d13b945
